_posts/2024-01-02-opensearch-ai-retrospective.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title:  "OpenSearch AI and machine learning: Recapping 2023 and what’s next"
+authors:
+ - dylatong
+date: 2024-01-02
+categories:
+ - community-updates, technical-posts, roadmap
+meta_keywords: vector database, vector search, ai, artificial intelligence, machine learning, neural-search, ml-commons, k-nn, ml-framework
```

Comment:
Please update the meta keywords and description with: 

**meta_keywords:** OpenSearch strategic enhancements, OpenSearch core engine, vector search engine, OpenSearch GenAI, machine learning framework

**meta_description:** Learn about strategic enhancements to the OpenSearch core engine since the Elasticsearch 7.10.2. fork and how they have improved the performance of log analytics and search use cases.